test/lib/browsers/playwright.ts

```diff
@@ -516,4 +517,10 @@ export class Playwright extends BrowserInterface {
       return page.waitForLoadState('networkidle')
     })
   }
+
+  locateRedbox(): Locator {
```

Comment:
This should really be in next-test-utils but exposing the whole page object may be a bit early.